velox/vector/VectorSaver.cpp

```diff
@@ -779,3 +779,10 @@ SelectivityVector restoreSelectivityVector(std::istream& in) {
 }
 
 } // namespace facebook::velox
+
+template <>
```

Comment:
because the actual encoding struct is wrapped in an anonymous namespace this formatter does not apply to it: https://github.com/facebookincubator/velox/actions/runs/7216170676/job/19661868123?pr=7941#step:6:1248

As the specialization can't be done within the namespace I am unsure what to do? rename the struct and remove the `namespace {`? Also not sure why this is not failing on all builds...

Reply:
nvm this is because it's the baseline